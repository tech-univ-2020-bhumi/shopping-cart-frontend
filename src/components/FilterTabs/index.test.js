import React from 'react';
import { render } from '@testing-library/react';
import FilterTab from './index';


describe('the FilterTab component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<FilterTab />);

    expect(asFragment()).toMatchSnapshot();
  });
});
