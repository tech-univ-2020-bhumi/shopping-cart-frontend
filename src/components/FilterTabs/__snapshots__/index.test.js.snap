// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the FilterTab component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <button
      data-testid="filter-button"
      type="button"
    >
      All
    </button>
  </div>
</DocumentFragment>
`;
